charts/synthetics-private-location/templates/deployment.yaml

```diff
@@ -33,16 +33,22 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.enableStatusProbes }}
```

Comment:
This mean we don't put a liveness probe by default anymore, is this the expected behavior? Or should we do an if/else to keep the old liveness probe when `enableStatusProbes` is disabled?

Reply:
Yes, it is the expected behavior 👍  We don't want to keep the old liveness probe as a fallback because it is deprecated, and it will be removed in the coming weeks/months.